src/index.js

```diff
@@ -28,6 +28,10 @@ export const ApolloClientToken: Token<ApolloClient> = createToken(
   'ApolloClientToken'
 );
 
+export const ApolloContextToken: Token<ApolloClient> = createToken(
```

Comment:
I am not sure if the flowtype for this is correct, given the usage of `typeof context === 'function' ? context(ctx) : context` in `fusion-apollo-universal-client`. Seems like this should be more specific, maybe either a function or mixed?

Reply:
Maybe this could also be a Fusion.js plugin, that way it can have dependencies on other things as well?